internal/config/ngt.go

```diff
@@ -57,6 +57,18 @@ type NGT struct {
 
 	// EnableInMemoryMode enables on memory ngt indexing mode
 	EnableInMemoryMode bool `yaml:"enable_in_memory_mode" json:"enable_in_memory_mode"`
+
+	// DefaultPoolSize represent default create index batch pool size
+	DefaultPoolSize uint32 `yaml:"default_pool_size" json:"default_pool_size"`
```

Comment:
Is this setting required? I feel that it may be set to defaultOption.

Reply:
same as below